### DISCUSSION TITLE
Create a NextAuth Discord or Slack community

### DISCUSSION THREAD

**Discussion by jack-pallot** [nextauthjs/next-auth#1559] [Ideas]
Since Auth can be a complex topic with many approaches, I think it would be beneficial to form an official NextAuth Discord or Slack community. Over time users can help to hopefully reduce the number of "how to" questions that can be answered in a more trivial format, and not have to rely as much on Github Issues and Discussions. I have seen other communities do this successfully and it's often the best place to go and get questions answered. Thoughts?

**Comment by balazsorban44** [comment]
To be honest, I think github discussions is the perfect place, since resolved issues/questions are searchable by anyone. No need to answer the same thing over and over. Although in practice, I admit people like to ignore searching themselves and just ask it again. Which from a maintainer's viewpoint is a bit sad 😔. I also think that we could appoint people from the community whose task wouldn't be to contribute to the code, but rather tighten the bond with the community by answering/managing issues and discussions here. Creating tutorials would also benefit much more people than easily lost discord/slack discussions in my opinion. It's just another wall we set up for those who just want an easy/quick answer.

We should rather educate people on how to interact with open source. No one should be afraid of searching problems in the source code (it's open!)  and maybe even open PRs with fixes. Personally if I as a maintainer would have to manage another channel would be a burden that I had to deal with in my free time.


Before GitHub Discussions, there was spectrum where a lot of communities were formed for the same reason. Github discussions is a good response to that. It won't send users to yet another platform.

Documentation improvements are also something that would just eliminate a lof of questions that are being asked. We welcome any PR that tries to clarify things up where our docs isn't perfect yet. when people want to form channels like this, it almost feels like we haven't done a great job explaining something in the docs that should be trivial. If a usecase is such a specific one, just open an issue with sufficient explanation/reproduction and you will get help from the maintainers/other users in a way that is visible to anyone else ending up in the same situation. 


All that said, I have nothing against someone from the community take it on themselves and create a discord/slack channel, but I don't think I want to have anything to do with it because of all the above mentioned reasons.

**Comment by jack-pallot** [comment]
Understood, thanks for taking the time to write a detailed reply.

**Reply by balazsorban44** [reply]
Thanks for understanding! Feel free to come with additional ideas how we can help our community better, either in the form of docs improvements or anything else, really!

**Comment by shadiramadan** [comment]
The problem with a rapidly changing project and github discussions is everything is out of date! :(
